configs/db.config.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose')
+
+const connectdb = async () => {
+    try {
+        const conn = await mongoose.connect(process.env.mongo_URI, {
+            useNewUrlParser: true,
```

Comment:
[Mongoose 6 always behaves as if useNewUrlParser, useUnifiedTopology, and useCreateIndex are true, and useFindAndModify is false. Please remove these options from your code.](https://mongoosejs.com/docs/migrating_to_6.html#no-more-deprecation-warning-options)

Reply:
This code is duplicate. You should only have db related configs here, connection code should reside in connection.js

Reply:
Thanks for the review sir, I correct them and then resend, thanks again sir